docs/cn/core-services/Caching.md

```diff
@@ -1,259 +1,507 @@
-* `alluxio.worker.tieredstore.level1.watermark.low.ratio=0.7` 设置了第二层的低水位比例0.7
+要从本地文件系统加载数据，使用命令
+[`alluxio fs copyFromLocal`]({{ '/en/operation/User-CLI.html' | relativize_url}}#copyfromlocal)。
+这只会将文件加载到Alluxio存储中，而不会将数据持久保存到UFS中。
+将写入类型设置为`MUST_CACHE`写入类型将不会将数据持久保存到UFS，
+而设置为`CACHE`和`CACHE_THROUGH`将会持久化保存。不建议手动加载数据，因为，当首次使用文件时Alluxio会自动将数据加载到Alluxio缓存中。
 
-定义存储层时有一些限制。Alluxio对于层级数量不做限制，首先一般是三层——内存，HDD和SDD。最多只有一层可以引用指定的别名。例如：最多有一层可以使用别名HDD。如果想在HDD层使用多个硬盘驱动器，可以配置`alluxio.worker.tieredstore.level{x}.dirs.path`为多个存储路径。
+###在Alluxio中持久化保留数据
 
-### 分层存储的参数配置
+命令[`alluxio fs persist`]({{ '/en/operation/User-CLI.html' | relativize_url}}#persist)
+允许用户将数据从Alluxio缓存推送到UFS。
 
-分层存储配置参数如下：
+```console
+$ ./bin/alluxio fs persist ${PATH_TO_FILE}
+```
 
-<table class="table table-striped">
-<tr><th>参数</th><th>缺省值</th><th>描述</th></tr>
-{% for item in site.data.table.tiered-storage-configuration-parameters %}
-<tr>
-<td>{{ item.parameter }}</td>
-<td>{{ item.defaultValue }}</td>
-<td>{{ site.data.table.cn.tiered-storage-configuration-parameters[item.parameter] }}</td>
-</tr>
-{% endfor %}
-</table>
+如果您加载到Alluxio的数据不是来自已已配置的UFS，则上述命令很有用
+。在大多数情况下，用户不必担心手动来持久化保留数据。
 
-## 设置生存时间 (TTL)
+###设置生存时间(TTL)
 
-Alluxio具有与每个文件或目录相关联的TTL属性。这些属性通过日志持久化。保证集群重启后的一致性。
-当Alluxio运行时，活跃的master节点负责保存元数据在内存中。在内部，master进程运行一个后台线程定期检查文件是否已经到达它对应的TTL值。
+Alluxio支持命名空间中每个文件和目录的”生存时间(TTL)”设置。此
+功能可用于有效地管理Alluxio缓存，尤其是在严格
+保证数据访问模式的环境中。例如，如果对上一周提取数据进行分析，
+则TTL功能可用于明确刷新旧数据，从而为新文件释放缓存空间。
 
-注意，后台线程在一个可配置的时间段内运行，默认为1小时。这意味着TTL在下一次检查间隔前不会强制执行，TTL的强制执行可以达到1
-TTL间隔延迟。间隔长度由 `alluxio.master.ttl.checker.interval` 属性设置。
+Alluxio具有与每个文件或目录关联的TTL属性。这些属性将保存为
+日志的一部分，所以集群重新后也能持久保持。活跃master节点负责
+当Alluxio提供服务时将元数据保存在内存中。在内部，master运行一个后台
+线程，该线程定期检查文件是否已达到其TTL到期时间。
 
-例如，将间隔设置为10分钟，将以下内容添加到 `alluxio-site.properties`:
+注意，后台线程按配置的间隔运行，默认设置为一个小时。
+在检查后立即达到其TTL期限的数据不会马上删除，
+而是等到一个小时后下一个检查间隔才会被删除。
 
-```
+如将间隔设置为10分钟，在`alluxio-site.properties`添加以下配置：
+
+```properties
 alluxio.master.ttl.checker.interval=10m
 ```
 
-查看[配置文档]({{ '/cn/operation/Configuration.html' | relativize_url }})获取有关Alluxio设置的更多配置
+请参考[配置页]({{ '/en/operation/Configuration.html' | relativize_url }})
+CN以获取有关设置Alluxio配置的更多详细信息。
 
-虽然master节点负责执行TTL，但是TTL值的设置取决于客户端
+#### API
 
-### 接口
+有两种设置路径的TTL属性的方法。
 
-有三种方法可以设置Alluxio的TTL。
-1. 通过Alluxio的shell命令行。
-1. 通过Alluxio Java文件系统API。
-1. 被动加载元数据或创建文件
+1.通过Alluxio shell命令行
+1.每个元数据加载或文件创建被动设置
 
 TTL API如下:
 
 ```
-SetTTL(path, duration, action)
-`path`          Alluxio命名空间中的路径
-`duration`      TTL操作生效之前的毫秒数，将覆盖任何之前的值
-`action`        生存时间过后要采取的行动。`FREE`将导致文件被逐出Alluxio存储，不管pin状态如何。
-                `DELETE`将导致文件从Alluxio命名空间中删除，并在存储中删除。
-                注意:`DELETE`是某些命令的默认值，将导致文件被永久删除。
+SetTTL(path，duration，action)
+`path` 		Alluxio命名空间中的路径
+`duration`	TTL动作生效前的毫秒数，这会覆盖任何先前的设值
+`action`	生存时间过去后要执行的`action`。 `FREE`将导致文件
+                从Alluxio存储中删除释放，无论其目前钉状态如何。 `DELETE`将导致
+                文件从Alluxio命名空间和底层存储中删除。
+                注意:`DELETE`是某些命令的默认设置，它将导致文件被
+                永久删除。
```

Comment:
descriptions better to be started at nearly the same column.
Take the original code block as an example